src/main/java/dinkplugin/util/Utils.java

```diff
@@ -177,7 +177,8 @@ public boolean containsEither(@NotNull String a, @NotNull String b) {
      * @return the name of the local player
      */
     public String getPlayerName(Client client) {
```

Comment:
Is it useful to tag this function as `Nullable`?

Reply:
Yeah I'll do a more complete audit of null safety since all of our notifiers only trigger in game, but we could be requested to dink when not logged in (unless we want to block these)